postgres/manage.sls

```diff
@@ -34,6 +36,7 @@ postgres-reload-modules:
 # Tablespace states
 
 {%- for name, tblspace in postgres.tablespaces|dictsort() %}
+ {%- do tblspace.update({'name': name}) %}
```

Comment:
required because I removed the automatic generation of name parameter from `format_state` macro